docs/testing/general.rst

```diff
@@ -11,7 +11,7 @@ interactions.
 This testing infrastructure includes them both with `Jasmine
 <http://jasmine.github.io/>`_ as the test suite and `Karma
 <http://karma-runner.github.io/>`_ as the test runner for unit tests.
-`Protractor <protractortest.org>`_ serves as the integration tests framework.
+`Protractor <http://angular.github.io/protractor>`_ serves as the integration tests framework.
```

Comment:
why not ``protractortest.org`` which is a more generic domain? WE'd have to add ``http://``